spark/src/test/scala/io/delta/sql/parser/DeltaSqlParserSuite.scala

```diff
@@ -274,8 +277,16 @@ class DeltaSqlParserSuite extends SparkFunSuite with SQLHelper {
     new UnresolvedAttribute(colName)
   }
 
-  private def tblId(tblName: String, schema: String = null): TableIdentifier = {
-    if (schema == null) new TableIdentifier(tblName)
-    else new TableIdentifier(tblName, Some(schema))
+  private def tblId(
+      tblName: String,
+      schema: String = null,
+      catalog: String = null): TableIdentifier = {
+    if (catalog == null) {
+      if (schema == null) new TableIdentifier(tblName)
+      else new TableIdentifier(tblName, Some(schema))
+    } else {
+      assert(schema != null)
+      new TableIdentifier(tblName, Some(schema), Some(catalog))
+    }
```

Comment:
Isn't this just:
```suggestion
    new TableIdentifier(tblName, Option(schema), Option(catalog))
```
`TableIdentifier` constructor already verifies that schema is non-null when catalog is non-null:
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/identifiers.scala#L101

Reply:
@ryan-johnson-databricks +1. We can update this later.